### ISSUE TITLE
isTimeout does not persist beyond a single request

### ISSUE THREAD

**Issue by benzittlau** [jakerella/jquery-mockjax#254]
Tracked the behaviour down to this line:
https://github.com/jakerella/jquery-mockjax/commit/419b28b0422c7cc93a4e7cf6b946071be7c6ba99#diff-53d3b5a93840121c909d8bea764b9e0eR470

I am attempting to test against network failures in my test suite.  I used mockjax's "isTimeout" feature to try and get all my requests to timeout:

```
    offlineMock = $.mockjax({
      url: '*',
      responseTime: 10,
      isTimeout: true
    });
```

Unfortunately because of the above line it will only timeout the *first* request, and then handle every other request with a 200 status code and an empty body.  I'm not sure why the linked line was added, but in my case removing it provides the desired behaviour.

**Comment by jakerella** [comment]
Hmm... good eye on that one. My guess is that we reset the `isTimeout` setting on the mock because we add it to the original settings for the xhr object itself, and maybe there is some conflict if we don't?

@dcneiner Can you take a look at the above referenced line and confirm why it's there or submit a fix otherwise?

**Comment by jakerella** [comment]
Should be fixed, it'll be in the 2.0.0 (non-beta) release.
